src/cswinrt/code_writers.h

```diff
@@ -4060,20 +4060,6 @@ R"(file static class %InterfaceEntriesImpl
 )", name, name, name, name);
     }
 
-    void write_winrt_typemapgroup_assembly_attribute(writer& w, TypeDef const& type)
```

Comment:
You do need to call this somewhere right to emit typemap or are you doing it elsewhere?

Reply:
ahh wait yea these are required for non-blittable types?